src/Options/index.js

```diff
@@ -103,6 +103,12 @@ export interface ParseServerOptions {
   databaseOptions: ?DatabaseOptions;
   /* Adapter module for the database; any options that are not explicitly described here are passed directly to the database client. */
   databaseAdapter: ?Adapter<StorageAdapter>;
+  /* Disable case insensitivity (collation) on queries and indexes, needed if you use MongoDB serverless or AWS DocumentDB. */
+  disableCollation: ?boolean;
```

Comment:
```suggestion
  /* Optional. If set to `true`, the collation rule of case comparison for queries and indexes is enabled. Enable this option to run Parse Server with MongoDB Serverless or AWS DocumentDB. If `false`, the collation rule of case comparison is disabled. Default is `false`.
  :DEFAULT: false */
  enableCollationCaseComparison: ?boolean;
```

- The term "disable collation" does not have a clear meaning. A "collation" is a set of rules for how to compare strings. Therefore, collation is never disabled, its just set to be case sensitive or insensitive. Also, to "disable case insensitivity" is a double negation, which makes it more difficult to understand than simply "enable case comparison".
- renamed to `enableCollationCaseComparison` so we can easier group future collation related options `enableCollation...`.